src/turnrest.c

```diff
@@ -165,7 +165,11 @@ janus_turnrest_response *janus_turnrest_request(const char *user) {
 	JANUS_LOG(LOG_VERB, "Sending request: %s\n", request_uri);
 	janus_mutex_unlock(&api_mutex);
 	curl_easy_setopt(curl, CURLOPT_URL, request_uri);
-	curl_easy_setopt(curl, api_http_get ? CURLOPT_HTTPGET : CURLOPT_POST, 1);
+	if(api_http_get){
+	        curl_easy_setopt(curl, CURLOPT_HTTPGET, 1L);
+	}else{
+		curl_easy_setopt(curl, CURLOPT_POST, 1L);
+	}
```

Comment:
Small editorial notes: per our code style, you need a space between parentheses, so

    if(api_http_get) {

and the else needs spaces too:

    } else {

Reply:
Thanks for the review and suggestions. Probably no longer needed.